RELEASE_NOTES.md

```diff
@@ -24,6 +24,77 @@ Chef now ships with Ruby 2.6.2. This new version of Ruby improves performance an
 
 ## Breaking Changes
 
+### Knife Bootstrap
+
+Knife bootstrap has been updated, and Windows bootstrap has been merged in Knife core bootstrap. This marks the deprecation of knife-windows plugin's `bootstrap` behavior.
+This addresses [CVE-2015-8559](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-8559): The knife bootstrap command in chef leaks the validator.pem private RSA key to /var/log/messages.
+
+In order to accomodate a combined bootstrap that supports both SSH and WinRM,
+CLI flags have been added/removed/changed.  Using the changed options will
+result in deprecation warnings, but will accept those options unless otherwise noted.
+Using removed options will cause the command to fail.
+
+#### New Flags
+
+| Flag | Description |
+|-----:|:-=----------|
```

Comment:
This "New Flags" chart is not displaying correctly. Is it because of this line's formatting?